charts/huawei-csi-plugin/values.yaml

```diff
@@ -0,0 +1,166 @@
+    resources: {}
+csi_node_driver_registrar:
+  image:
+    # csi_node_driver_registrar.image.repository -- Image Repo for csi-node-driver-registrar
+    repository: "quay.io/k8scsi/csi-node-driver-registrar"
+    # csi_node_driver_registrar.image.tag -- Image Tag for csi-node-driver-registrar
+    tag: "v1.2.0"
+    # csi_node_driver_registrar.image.pullPolicy -- Image PullPolicy for csi-node-driver-registrar
+    pullPolicy: IfNotPresent
+  # csi_node_driver_registrar.resources -- resources for csi-node-driver-registrar
+  resources: {}
+csi_provisioner:
+  image:
+    # csi_provisioner.image.repository -- Image Repo for csi-provisioner
+    repository: "quay.io/k8scsi/csi-provisioner"
+    # csi_provisioner.image.tag -- Image Tag for csi-provisioner
+    tag: "v1.5.0"
+    # csi_provisioner.image.pullPolicy -- Image PullPolicy for csi-provisioner
+    pullPolicy: IfNotPresent
+  # csi_provisioner.resources -- resources for csi-provisioner
+  resources: {}
+csi_resizer:
+  image:
+    # csi_resizer.image.repository -- Image Repo for csi-resizer
+    repository: "quay.io/k8scsi/csi-resizer"
+    # csi_resizer.image.tag -- Image Tag for csi-resizer
+    tag: "v0.5.0"
+    # csi_resizer.image.pullPolicy -- Image PullPolicy for csi-resizer
+    pullPolicy: IfNotPresent
+  # csi_resizer.resources -- resources for csi-resizer
+  resources: {}
+csi_snapshot_controller:
+  image:
+    # csi_snapshot_controller.image.repository -- Image Repo for csi-snapshot-controller
+    repository: "quay.io/k8scsi/csi-snapshot-controller"
+    # csi_snapshot_controller.image.tag -- Image Tag for csi-snapshot-controller
+    tag: "v2.1.0"
+    # csi_snapshot_controller.image.pullPolicy -- Image PullPolicy for csi-snapshot-controller
+    pullPolicy: IfNotPresent
+  # csi_snapshot_controller.resources -- resources for csi-snapshot-controller
+  resources: {}
+csi_snapshotter:
+  image:
+    # csi_snapshotter.image.repository -- Image Repo for csi-snapshotter
+    repository: "quay.io/k8scsi/csi-snapshotter"
+    # csi_snapshotter.image.tag -- Image Tag for csi-snapshotter
+    tag: "v2.1.0"
+    # csi_snapshotter.image.pullPolicy -- Image PullPolicy for csi-snapshotter
+    pullPolicy: IfNotPresent
+  # csi_snapshotter.resources -- resources for csi-snapshotter
+  resources: {}
+
+# imagePullSecrets -- specifies the image pull secrets to be used
+imagePullSecrets: []
+# nameOverride -- specifies the name override to be used for helm
+nameOverride: ""
+# fullnameOverride -- specifies the full name override to be used for helm
+fullnameOverride: ""
+
+serviceAccount:
+  # serviceAccount.create -- Whether to create serviceAccounts or not
+  create: true
+  # serviceAccount.annotations -- annotations to add to each service account
+  annotations: {}
+  # serviceAccount.controllerName -- The name of the service account for the controller account
+  controllerName: "huawei-csi-controller"
```

Comment:
Would be cool if this would have some sort of stanadard like `{{ template "filebeat.fullname" . }}-controller` or `{{ template "filebeat.fullname" . }}-node`  for being able to not have to define a "default" here.

Reply:
done :white_check_mark: